airflow/decorators/__init__.pyi

```diff
@@ -14,12 +14,12 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
```

Comment:
do you have pre-commit enabled? i would not expect a change to this line

Reply:
yes, I didn't change this line, but the hook `Run black (python formatter)` did